code/datums/uplink_item.dm

```diff
@@ -313,6 +313,14 @@ var/list/uplink_items = list()
 	cost = 6
 	job = list("Internal Affairs Agent")
 
+//Roboticist
+/datum/uplink_item/jobspecific/remotecyborg_kit
+	name = "Remote Cyborg Kit"
+	desc = "A box including everything you need to make a remote cyborg, plus a couple of cyborg cameras."
+	item = /obj/item/weapon/storage/box/remotecontrolkit
+	cost = 6
```

Comment:
Unacceptably low. Also, make it impossible to emag. Full control over an emagged borg is much too powerful a tool to be given out.